Card/Statistics.ts

```diff
@@ -0,0 +1,16 @@
+import { isoly } from "isoly"
+import { Scheme } from "./Scheme"
+
+export namespace Statistics {
+	export type Request = {
+		scheme: Scheme
+		range: isoly.DateRange
+		options?: { limit: number; cursor?: string }
+	}
+	export type Response = {
+		new: number
+		old: number
+		withTransaction: number
```

Comment:
I think these property names are a bit unclear, could `new` and `old` be called eg. `atStart` and `atEnd`?